include/maxminddb.h

```diff
@@ -161,6 +161,7 @@ typedef struct MMDB_ipv4_start_node_s {
 } MMDB_ipv4_start_node_s;
 
 typedef struct MMDB_s {
+    bool file_mapping;
```

Comment:
Won't this be an ABI change? Maybe we should set the open type with a flag instead.